module/vision/GoalDetector/src/GoalDetector.cpp

```diff
@@ -239,6 +250,7 @@ namespace vision {
                         }
                     }
 
+
```

Comment:
Try to avoid putting random newlines in when staging your changes to commit, they are just asking for conflicts later.